lib/workers/pr/changelog/release-notes.ts

```diff
@@ -179,14 +179,15 @@ export async function getReleaseNotesMd(
     if (changelogParsed.length >= 2) {
       for (const section of changelogParsed) {
         try {
-          const [heading] = section.split('\n');
+          const [heading] = section.replace(/[[\]()]/g, ' ').split('\n');
```

Comment:
We need to make this more understandable for anyone in future reading it. This can be done either with comments, or by breaking it up into multiple steps with meaningful variable names, or a combination of both. Of course, tests help too.

Reply:
Sure, will do that